lib/runner.ts

```diff
@@ -258,9 +260,13 @@ export class Runner extends EventEmitter {
         newBrowser.mockModules_ = browser_.mockModules_;
       }
       if (opt_useSameUrl) {
-        browser_.driver.getCurrentUrl().then((url: string) => {
-          newBrowser.get(url);
-        });
+        newBrowser.ready = wdpromise.all([newBrowser.ready, browser_.driver.getCurrentUrl()])
```

Comment:
This seems simpler as:

```ts
newBrowser.ready = newBrowser.ready().then(() => {
  return browser_.driver.getCurrentUrl();
}).then((url) => {
  return newBrowser.get(url);
}).then(() => {
  return newBrowser;
});
```
instead of messing with the promise.all.